src/authentication/accept-invite/index.tsx

```diff
@@ -42,11 +80,15 @@ export class Invite extends React.Component<Properties, State> {
           type='number'
         />
 
+        {this.state.showAlert &&
```

Comment:
Actually, I think the Input box has alert message capability:
```
alert = { variant : "error", text : "Something is  wrong"}
```

Is that more appropriate here since it's always related to the invite code?...or is it more generic than that?

Reply:
as discussed on standup call, will keep the state. 

> I'd also rename showAlert to renderAlert so that it's more clear that it's a rendering method. Just a conventional thing I've picked up.

Thanks for suggestion, will do.